pyvista/utilities/geometric_objects.py

```diff
@@ -465,6 +466,36 @@ def Line(pointa=(-0.5, 0.0, 0.0), pointb=(0.5, 0.0, 0.0), resolution=1):
     return line
 
 
+def MultipleLines(points=[[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]]):
+    """Create multiple lines.
+
+    Parameters
+    ----------
+    points : np.ndarray or list, optional
+        List of points defining a broken line, default is [[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]].
```

Comment:
Should [[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]] have double back ticks?  I'm not sure.

Reply:
```suggestion
        List of points defining a broken line, default is ``[[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]]``.
```